.github/workflows/kakarot_rpc.yml

```diff
@@ -109,3 +109,37 @@ jobs:
       - name: Inspect image
         run: |
           docker buildx imagetools inspect ${{ env.REGISTRY_IMAGE }}:latest
+
+  build-hive:
+    runs-on: ubuntu-latest
+    # Latest Kakarot RPC image is required to build Hive RPC image
+    needs:
+      - merge
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+      - name: Retrieve cached dependencies
+        uses: Swatinem/rust-cache@v2
+      - name: Dump Genesis
+        run: make dump-genesis
```

Comment:
This step is needed before we build the image so that the combined `genesis.json` will exist